sdk/go/pulumi-language-go/language_test.go

```diff
@@ -172,8 +172,9 @@ func runTestingHost(t *testing.T) (string, testingrpc.LanguageTestClient) {
 
 // Add test names here that are expected to fail and the reason why they are failing
 var expectedFailures = map[string]string{
-	"l2-plain":    "cannot use &plain.DataArgs{…} (value of type *plain.DataArgs) as plain.DataArgs value in struct literal", //nolint:lll
-	"l2-map-keys": "cannot use &plain.DataArgs{…} (value of type *plain.DataArgs) as plain.DataArgs value in struct literal", //nolint:lll
+	"l2-plain":                  "cannot use &plain.DataArgs{…} (value of type *plain.DataArgs) as plain.DataArgs value in struct literal", //nolint:lll
+	"l2-map-keys":               "cannot use &plain.DataArgs{…} (value of type *plain.DataArgs) as plain.DataArgs value in struct literal", //nolint:lll
+	"l2-parameterized-resource": "go mod tidy fails",
```

Comment:
can you explain why? Is there something we need to do to fix it?

Reply:
I don't know why yet, need to investigate it still. Go has had a bunch of these issues before. It is very particular in how it references local package